server/game/game.js

```diff
@@ -75,11 +75,11 @@ class Game extends EventEmitter {
         this.skipPhase = {};
         this.cardVisibility = new CardVisibility(this);
 
-        for(let player of details.players || []) {
+        for(let player of Object.values(details.players || {})) {
```

Comment:
Seems fairly unrelated to this change?

Reply:
It is, but the game node server won’t run without this being fixed :)